.travis.yml

```diff
@@ -35,6 +35,7 @@ matrix:
     - env: TEST_SPECIFIC_MODULES=presto-redis
     - env: TEST_SPECIFIC_MODULES=presto-sqlserver,presto-postgresql,presto-mysql
     - env: TEST_SPECIFIC_MODULES=presto-phoenix,presto-iceberg
+    - env: TEST_SPECIFIC_MODULES=presto-oracle
```

Comment:
The CI configuration has moved to `.github/workflows/ci-tests.yml`

Reply:
changed